src/main/java/org/rus4j/numbify/lang/ru/Russian.java

```diff
@@ -30,6 +30,22 @@ public Russian(RuDeclension declension, Gender[] genders, Currency currency, Str
         this.decimalSeparator = decimalSeparator;
     }
 
+    /**
+     * 'Russian' constructor with default genders.
+     * By default, genders[0] = Gender.MALE, genders[1] = Gender.FEMALE.
+     */
+    public Russian(RuDeclension declension, Currency currency, String decimalSeparator) {
+        this(declension, new Gender[] {Gender.MALE, Gender.FEMALE}, currency, decimalSeparator);
```

Comment:
Gender array has to be calculated based on `currency`. See `currencyGender(currency)`